arbnode/node.go

```diff
@@ -414,6 +414,9 @@ type Config struct {
 }
 
 func (c *Config) Validate() error {
+	if !c.DelayedSequencer.Enable && c.Sequencer.Enable {
+		log.Warn("sequencer and l1 reader, without delayed sequencer")
```

Comment:
The message suggests that it'd only log if the L1 reader also is enabled, which seems like a good idea but isn't in the if statement.